client/src/components/Instructions/index.js

```diff
@@ -270,6 +364,77 @@ export default class Instructions extends Component {
     );
   }
 
+    renderGaslessUpgrade() {
+      return (
+        <div className={styles.instructions}>
+          <h2> Upgrading your contract with access control </h2>
+          <p> Thanks to ZeppelinOS, you can upgrade the code of your contract to add more functionality. With this functionality we can restrict access to relayed transactions. </p>
```

Comment:
Should we add an explanation on what this means? So far, we haven't told the user what a "relayed transaction" is. I think it is mentioned in the component's README, but it's easy for the user to miss that.

Reply:
That's a good idea, I will add a section on relayed transactions when the user is upgrading the contract